### DISCUSSION TITLE
Is there a way to collaborate with a friend?

### DISCUSSION THREAD

**Discussion by jreoka** [omnivore-app/omnivore#3589] [Q&A] [answered]
This would be a nice feature!

**Comment by IamSmeagol** [comment, marked as answer]
https://docs.omnivore.app/about/pricing.html
> We have a few product ideas we have experimented with that would be paid add-ons to the current service: **collaborative tools**, AI integration, translation tools, and premium text to speech voices. Of these features, the premium text to speech voices are the only one that are currently available in the app, as part of our "ultra realistic voices" beta.

Based on this, that feature has been considered, but may  be under a premium plan
